pom.xml

```diff
@@ -3,6 +3,37 @@
   <groupId>com.demo.locustapitest</groupId>
   <artifactId>locustapitest</artifactId>
   <version>0.0.1-SNAPSHOT</version>
+  <name>locust4j</name>
+  <description>LocustApiTest is a load automation framework test written in java that use cucumber, restassured and locust4j.</description>
+  <url>https://github.com/dmarcas/locustapitest.git</url>
+  
+  <licenses>
+       <license>
+           <name>MIT License</name>
+           <url>http://www.opensource.org/licenses/mit-license.php</url>
+           <distribution>repo</distribution>
+       </license>
+   </licenses>
+   
+   <issueManagement>
+       <system>Github Issues</system>
+       <url>https://github.com/dmarcas/locustapitest/issues</url>
+   </issueManagement>
+   
+   <developers>
+        <developer>
+            <name>dmarcas</name>
+            <id>dmarcas</id>
+            <email>david.marcas@gmail.com</email>
+            <url>https://github.com/dmarcas</url>
+        </developer>
+        <developer>
+            <name>Andrei Danilov</name>
+            <id>therussiankid92</id>
+            <email>andreialexandr.danilov@gmail.com</email>
```

Comment:
Puedes quitar mi correo? No quiero que me lo parseen los bots xD

Reply:
of course. I just remove mine too ...